### PR TITLE
Calculate and Operate tests

### PR DESCRIPTION
**Pull request by alex1779**
In this PR we:

- added calculate and operate tests
- fix linter errors
- add snapshop test for home component

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have descriptive PR description. Please add a short summary of changes made in this PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/tests/opertate.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import operate from '../logic/operate';
+
+const numberOne = 2;
+const numberTwo = 4;
+const operation = 'x';
+
+test('return the result of multiplication of two numbers', () => {
+  expect(operate(numberOne, numberTwo, operation)).toEqual('8');
+});
```
Comment (roshan-bajgain) [inline src/tests/opertate.test.js]: - [x] If you see the project requirement then you should test all the functionalities & scenarios of operate.js that is `+,-,*,/,%`. etc. Please write the test for all the functionality.

File: src/tests/home.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import React from 'react';
+import Home from '../components/Home';
+
+test('Home component snapshot', () => {
+  const component = renderer.create(<Home />);
+  const tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (roshan-bajgain) [inline src/tests/home.test.js:9]: - [x] As the project requirement you should create unit tests for all the react components. So please create tests for all other components as well. You did not test all the component that is in the component folder.

File: src/tests/calculate.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import calculate from '../logic/calculate';
+
+test('clear display when button is clicked', () => {
+  const buttonName = 'AC';
+  const objNull = {
+    total: null,
+    next: null,
+    operation: null,
+  };
```
Comment (roshan-bajgain) [inline src/tests/calculate.test.js]: - [x] There are other functionality in the calculate.js file please make sure to test that functionality as well.
- [x] when clicking digits (0-9),  when clicking the decimal point ('.'), when clicking the '=' button to perform a calculation, please write all the test.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @alex1779 and @Estete9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Some test are passing ✔️ 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @alex1779  
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All test passed
- [ ] all methods tested and compoenets tested

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**